lib/provider.js

```diff
@@ -115,7 +116,7 @@ class Provider extends events.EventEmitter {
 
     super();
 
-    this.issuer = issuer;
```

Comment:
Why are you removing the trailing slash in the issuer identifier?

Reply:
In tests I was getting some failures because a route had double slashes, which didn't match the actual/expected value which had a single slash.

I thought it would be nice to ensure everything would always have a trailing slash removed. But I'm happy to put this back how it was and maybe update how the paths are generated elsewhere